sesman/config.c

```diff
@@ -212,6 +212,7 @@ config_read_security(int file, struct config_security *sc,
     sc->ts_admins_enable = 0;
     sc->restrict_outbound_clipboard = 0;
     sc->restrict_inbound_clipboard = 0;
+    sc->allowed_alternate_shells = 0;
```

Comment:
Might as well set the default here, rather than using 0 here and checking for it later on:-

```c
sc->allowed_alternate_shells = g_strdup("YES");
```